modules/core/share_limits.py

```diff
@@ -210,7 +213,7 @@ def update_share_limits_for_group(self, group_name, group_config, torrents):
                 check_max_ratio or check_max_seeding_time or check_limit_upload_speed or share_limits_not_yet_tagged
             ) and hash_not_prev_checked:
                 if not is_tag_in_torrent(MIN_SEEDING_TIME_TAG, torrent.tags) and not is_tag_in_torrent(
-                    MIN_NUM_SEEDS_TAG, torrent.tags
+                    MIN_NUM_SEEDS_TAG, torrent.tags and not is_tag_in_torrent(LAST_ACTIVE_TAG, torrent.tags)
                 ):
```

Comment:
Isn't the ending bracket misplaced?


```suggestion
                if not is_tag_in_torrent(MIN_SEEDING_TIME_TAG, torrent.tags) and not is_tag_in_torrent(
                    MIN_NUM_SEEDS_TAG, torrent.tags) and not is_tag_in_torrent(LAST_ACTIVE_TAG, torrent.tags
                ):
```

Reply:
Good point, it is.